addons/edi/models/edi_document.py

```diff
@@ -85,6 +85,11 @@ def _default_project_id(self):
     _sql_constraints = [('model_uniq', 'unique (model_id)',
                          "The document model must be unique")]
 
+    # optionally enforce document name globs
+    enforce_filename = fields.Boolean(string="Enforce document filenames follow glob pattern",
+                                  help="Enforce document filenames follow glob pattern",
+                                  default=True)
```

Comment:
Do product want `default=True`? I can see the potential for that to cause problems